M2/Macaulay2/m2/format.m2

```diff
@@ -62,7 +62,7 @@ info TITLE :=
 net  COMMENT :=
 info COMMENT :=
 net  LITERAL :=
-info LITERAL := x -> ""
+info LITERAL := x -> ()
```

Comment:
`info` is to return a net, so this can't be right.

Reply:
> info is to return a net, so this can't be right.

The typical value of `info` is String, perhaps that should be fixed. When combined in any other hypertext object, it still returns a Net:
```m2
i9 : info DIV{"a", LITERAL "b", "c"}

o9 = ac

i10 : class oo

o10 = Net

o10 : Type

i11 : peek ooo

      +--+
o11 = |ac|
      +--+
```

Here is the reason for the change:
old behavior:
```m2
i1 : R = kk[x,y];

i2 : info help R

o2 = R 
     **



     For the programmer
     ==================

     The object R (missing documentation) is a *note polynomial ring: (Macaulay2Doc)PolynomialRing,, with ancestor classes *note
     RingElement: (Macaulay2Doc)RingElement, < *note BasicList: (Macaulay2Doc)BasicList, < *note Thing: (Macaulay2Doc)Thing,.

i3 : net help R

o3 = R 
     **



     For the programmer
     ==================

     The object "R" is a polynomial ring, with ancestor classes "RingElement" < "BasicList" < "Thing".
```
New behavior:
```m2
i1 : R = kk[x,y];

i2 : info help R

o2 = R 
     **

     For the programmer
     ==================

     The object R (missing documentation) is a *note polynomial ring: (Macaulay2Doc)PolynomialRing,, with ancestor classes *note
     RingElement: (Macaulay2Doc)RingElement, < *note BasicList: (Macaulay2Doc)BasicList, < *note Thing: (Macaulay2Doc)Thing,.

i3 : net help R

o3 = R 
     **

     For the programmer
     ==================

     The object "R" is a polynomial ring, with ancestor classes "RingElement" < "BasicList" < "Thing".
```
Notice the extra line not being there.